.stylelintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": ["stylelint-config-standard"],
+  "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
+  "rules": {
+    "at-rule-no-unknown": null,
+    "scss/at-rule-no-unknown": true,
+    "csstree/validator": true
+  },
+  "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css", "**/*.js", "**/*.jsx"]
+}
```

Comment:
Please add this file to .gitignore and remove it from Github. Because this is a linter file just to check errors locally. Keeping unused files is not a good practice.

Reply:
Great advice! thank you :smiley: on it :thumbsup: